docs/index.md

```diff
@@ -40,6 +40,7 @@ things, PMD can be run:
 * As a [Maven goal](pmd_userdocs_tools_maven.html)
 * As an [Ant task](pmd_userdocs_tools_ant.html)
 * As a [Gradle task](https://docs.gradle.org/current/userguide/pmd_plugin.html)
+* As a [bld operation](https://github.com/rife2/bld-pmd)
```

Comment:
Since #4669 we have a own documentation -> https://docs.pmd-code.org/snapshot/pmd_userdocs_tools_bld.html

```suggestion
* As a [bld operation](pmd_userdocs_tools_bld.html)
```

I'll also change the link for gradle to point to https://docs.pmd-code.org/snapshot/pmd_userdocs_tools_gradle.html